templates/password.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * @author Jörn Friedrich Dreyer <jfd@butonic.de>
+ *
+ * @copyright Copyright (c) 2018, ownCloud GmbH
+ * @license GPL-2.0
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+script('password_policy', 'password');
+style('password_policy', 'styles');
+
+/**
+ * @var array $_
+ * @var \OCP\IL10N $l
+ * @var \OC_Defaults $theme
+ */
+?>
+
+<form id="password_policy" method="post">
+	<fieldset>
+	<h1 class="warning"><?php p($l->t('Password changed required'));?></h1>
+
+	<?php if (isset($_['error'])) { ?><div id="error" class="warning"><?php p($_['error']) ?></div> <?php } ?>
+
+	<p>
+
+	<input type="password" id="current_password" name="current_password"
+		   value="" autofocus placeholder="<?php p($l->t('current password'));?>"
+	/>
+	<label for="current_password" class="infield"><?php p($l->t('current password'));?></label>
+
+	<input type="password" id="new_password" name="new_password"
+		   value="" placeholder="<?php p($l->t('new password'));?>"
+	/>
+	<label for="new_password" class="infield"><?php p($l->t('new password'));?></label>
+
+	<input type="password" id="retyped_password" name="retyped_password"
+		   value="" placeholder="<?php p($l->t('retype new password'));?>"
```

Comment:
is *retyped* proper english? @settermjd

Reply:
Yes. However, "confirm new password" seems to be the defacto standard.